### PR TITLE
#774 : MediaGalleryUiApi module added

### PR DESCRIPTION
**Pull request by konarshankar07**
### Description (*)

This PR will introduce MediaGalleryUiApi module and also added service contract for `\Magento\MediaGalleryUi\Model\Config`
### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#774: [1.1-develop] Introduce MediaGalleryUiApi module and ConfigInterface

### CODE REVIEW

File: MediaGalleryUi/Model/Config.php [hunk 1]
```diff
@@ -9,11 +9,12 @@
 namespace Magento\MediaGalleryUi\Model;
 
 use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\MediaGalleryUiApi\Model\ConfigInterface;
```
Comment (konarshankar07) [inline MediaGalleryUi/Model/Config.php]: @sivaschenko .. I think we should move `ConfigInterface` from `Model` folder to `Api` folder

File: MediaGalleryUiApi/composer.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+    "name": "magento/module-media-gallery-ui-api",
+    "description": "Magento module responsible for the media gallery UI implementation API",
+    "require": {
+        "php": "~7.1.3||~7.2.0||~7.3.0",
+        "magento/framework": "*"
```
Comment (konarshankar07) [inline MediaGalleryUiApi/composer.json:6]: Do I need to add `MediaGalleryUi` module in require?

Comment (sivaschenko) [reply MediaGalleryUiApi/composer.json:6]: nope, it should be the other way around - MediaGalleryUi is supposed to implement interfaces of MediaGalleryUiApi - So MediaGalleryUiApi should be required for MediaGalleryUi module

File: MediaGalleryUi/Model/Config.php (hunk 1 above)
Comment (sivaschenko) [reply MediaGalleryUi/Model/Config.php]: Yes

**Review by sivaschenko — CHANGES_REQUESTED** [review]
Hi @konarshankar07 thanks for the pull request! Please correct the `ConfigInterface` namespace in the reference and update the MediaGalleryUi composer.json with MediaGalleryUiApi dependency

**Review by sivaschenko — APPROVED** [review]

**Pull request merged**